azure-cs-functions/FunctionsStack.cs

```diff
@@ -78,6 +78,10 @@ public FunctionsStack()
             {
                 AppSettings = new[]
                 {
+                    new NameValuePairArgs{
+                        Name = "AzureWebJobsStorage",
+                        Value = GetConnectionString(resourceGroup.Name, storageAccount.Name),
+                    },
```

Comment:
Unrelated to this change but I think it would be good to use a separate `WebAppApplicationSettings` resource which is very useful in practice

Reply:
Agreed. That said, we have some coverage here now: https://github.com/pulumi/examples/blob/master/azure-cs-credential-rotation-one-set/AppStack.cs#L249.